src/v/config/configuration.cc

```diff
@@ -609,6 +609,60 @@ configuration::configuration()
       std::nullopt)
   , superusers(
       *this, "superusers", "List of superuser usernames", required::no, {})
+  , kafka_qdc_latency_alpha(
+      *this,
+      "kafka_qdc_latency_alpha",
+      "Smoothing parameter for kafka queue depth control latency tracking.",
+      required::no,
+      0.002)
+  , kafka_qdc_window_size_ms(
+      *this,
+      "kafka_qdc_window_size_ms",
+      "Window size for kafka queue depth control latency tracking.",
+      required::no,
+      std::chrono::milliseconds(1500))
```

Comment:
the config uses literals `1.5sec`

Reply:
that doesn't seem to auto convert from float :(